### PR TITLE
Index google data

### LINKED ISSUE #7: Index Google Data
Part of #3.

Based on some categories, query google search, open the first 100 pages and index the content in the database.

### PR DESCRIPTION
**Pull request by tiriplicamihai**
Fixes #7

### CODE REVIEW

**Comment by tiriplicamihai** [comment]
@piatra can you please review?

File: src/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM python:2.7.11
+
+# Install packages.
+ADD requirements.txt .
+
+# This is here because we want to use caching. It takes a lot to download nltk data.
```
Comment (piatra) [inline src/Dockerfile:6]: 👍

File: src/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+version: '2'
+
+services:
+    datasetbuilder:
+        build: .
+        command: ["sleep", "100000000"]
```
Comment (piatra) [inline src/docker-compose.yml:6]: What's this for?

Comment (mihai-tiriplica-hs) [inline src/docker-compose.yml:6]: It's here just to keep the container up and running. We should upgrade this to the actual command but we still have to figure out how will we keep the database.

File: src/requirements.txt [hunk 3]
```diff
@@ -1,3 +1,5 @@
 scrapy==1.0.5
 SQLAlchemy
 beautifulsoup4>=4.0
+ipdb
```
Comment (piatra) [inline src/requirements.txt:4]: for what?

Comment (mihai-tiriplica-hs) [inline src/requirements.txt:4]: For debugging. It's pdb on steroids - opens the debugger in an ipython console with autocomplete and all the goodies.

File: src/models.py [hunk 4]
```diff
@@ -25,3 +28,12 @@ class Websites(DeclarativeBase):
...
     male_ratio = Column('male_ratio', Float, nullable=True)
     female_ratio = Column('female_ratio', Float, nullable=True)
+
+
+class WebsitesContent(DeclarativeBase):
+    """Sqlalchemy websites model"""
+    __tablename__ = "websites_content"
+
+    id = Column(Integer, primary_key=True)
+    link = Column('link', String, nullable=False)
```
Comment (piatra) [inline src/models.py:38]: Should this be unique?

**Pull request merged**